### PR TITLE
Updates/Bug fixes

### PR DESCRIPTION
**Pull request by lzchen**
- add metrics example to README
- change README to markdown file
- remove ikeys from examples
- add metrics example
- fix protocol classes
- update example files to reflect new OT changes

### CODE REVIEW

**Comment by hectorhdzg** [comment]
CI is back, awesome

File: azure_monitor/examples/traces/request.py [hunk 1]
```diff
@@ -6,19 +6,21 @@
...
 from opentelemetry import trace
 from opentelemetry.ext import http_requests
-from opentelemetry.sdk.trace import TracerSource
+from opentelemetry.sdk.trace import TracerProvider
 from opentelemetry.sdk.trace.export import SimpleExportSpanProcessor
 
 from azure_monitor import AzureMonitorSpanExporter
 
-trace.set_preferred_tracer_source_implementation(lambda T: TracerSource())
+trace.set_preferred_tracer_provider_implementation(lambda T: TracerProvider())
```
Comment (hectorhdzg) [inline azure_monitor/examples/traces/request.py]: OT code for this is not published yet, do we want to merge this right now or until beta is released?

**Review by hectorhdzg — APPROVED** [review]

File: azure_monitor/examples/traces/request.py (hunk 1 above)
Comment (lzchen) [reply azure_monitor/examples/traces/request.py]: Yeah I think that's why the build is failing. I will probably leave the logic as `tracer_source` until OT is published as to not have breaking code.

Comment (lzchen) [reply azure_monitor/examples/traces/request.py]: EDIT: Actually I'll probably comment on the examples for now until the OT beta is released.

**Pull request merged**